rskj-core/src/test/java/co/rsk/peg/whitelist/WhitelistStorageProviderImplTest.java

```diff
@@ -0,0 +1,114 @@
+        assertEquals(1, actualLockWhitelist.getAll().size());
+    }
+
+    @Test
+    void save_whenIsActiveRSKIP87_ShouldSavedUnlimitedLockWhitelist() {
+        when(activationConfig.isActive(ConsensusRule.RSKIP87)).thenReturn(true);
+        LockWhitelist lockWhitelist = whitelistStorageProvider.getLockWhitelist(activationConfig, networkParameters);
+        LockWhitelistEntry oneOffWhiteListEntry =  addOneOffWhiteListEntry();
+        LockWhitelistEntry unlimitedWhiteListEntry = addUnlimitedWhiteListEntry();
+        lockWhitelist.put(getBtcAddress(), oneOffWhiteListEntry);
+        lockWhitelist.put(getBtcAddress(), unlimitedWhiteListEntry);
+
+        whitelistStorageProvider.save(activationConfig);
+
+        LockWhitelist actualLockWhitelist = whitelistStorageProvider.getLockWhitelist(activationConfig, networkParameters);
+        assertEquals(2, actualLockWhitelist.getAll().size());
+    }
+
+    @Test
+    void getLockWhitelist_whenLockWhitelistIsNull_shouldReturnZeroEntries() {
+        LockWhitelist actualLockWhitelist = whitelistStorageProvider.getLockWhitelist(activationConfig, networkParameters);
+
+        assertEquals(0, actualLockWhitelist.getAll().size());
+    }
+
+    @Test
+    void getLockWhitelist_whenLockWhitelistIsNotNull_shouldReturnOneEntry() {
+        LockWhitelist lockWhitelist = whitelistStorageProvider.getLockWhitelist(activationConfig, networkParameters);
+        LockWhitelistEntry oneOffWhiteListEntry =  addOneOffWhiteListEntry();
+        lockWhitelist.put(getBtcAddress(), oneOffWhiteListEntry);
+
+        LockWhitelist actualLockWhitelist = whitelistStorageProvider.getLockWhitelist(activationConfig, networkParameters);
+
+        assertEquals(1, actualLockWhitelist.getAll().size());
+    }
+
+    @Test
+    void getWhitelist_whenIsActiveRSKIP87_shouldReturnTwoEntries() {
+        when(activationConfig.isActive(ConsensusRule.RSKIP87)).thenReturn(true);
+        LockWhitelist lockWhitelist = whitelistStorageProvider.getLockWhitelist(activationConfig, networkParameters);
+        LockWhitelistEntry oneOffWhiteListEntry =  addOneOffWhiteListEntry();
+        LockWhitelistEntry unlimitedWhiteListEntry = addUnlimitedWhiteListEntry();
+        lockWhitelist.put(getBtcAddress(), oneOffWhiteListEntry);
+        lockWhitelist.put(getBtcAddress(), unlimitedWhiteListEntry);
+
+        LockWhitelist actualLockWhitelist = whitelistStorageProvider.getLockWhitelist(activationConfig, networkParameters);
+
+        assertEquals(2, actualLockWhitelist.getAll().size());
+    }
+
+    private LockWhitelistEntry addOneOffWhiteListEntry() {
+        Coin maxTransferValue = networkParameters.getMaxMoney();
+        return new OneOffWhiteListEntry(getBtcAddress(), maxTransferValue);
+    }
+
+    private LockWhitelistEntry addUnlimitedWhiteListEntry() {
+        return new UnlimitedWhiteListEntry(getBtcAddress());
+    }
+
+    private Address getBtcAddress() {
+        BtcECKey btcECKey = new BtcECKey();
```

Comment:
We have a method for this in BitcoinTestUtils class, let's use that

Reply:
Done! Created the `BitcoinTestUtils.getBtcAddress` method to return a BTC address derived from a seed. I didn't use `getBtcEcKeysFromSeeds` method since it converts from an `array` of `seeds` to `List<BtcECKey>`. 
Additionally, I created 2 instance variables that get the value from this new method, to be used in different UTs.